src/routes.js

```diff
@@ -17,7 +17,7 @@ export default [
     meta: {
       feature: "Proposals"
     },
-    component: require(`./components/governance/PageProposals`).default
+    component: () => import(`./components/governance/PageProposals`)
```

Comment:
This is already loading the pages and there content async

Reply:
i don't think so. why do you think that?